src/common/options/global.yaml.in

```diff
@@ -2299,6 +2306,13 @@ options:
   level: advanced
   default: 3
   with_legacy: true
+# how many mons connections are we keeping in the aux list
+- name: mon_aux_list_connections
+  type: uint
+  level: advanced
+  default: 0
```

Comment:
no description? should it be a dev option? 0 means unlimited?

Reply:
i think it should be advanced, the user can decide how many open connections they accept for each monclient.
in case of too many open connections, they can reduce that parameter.
We are currently using 0 as the default to disable that feature. No aux list is in used.